jenkins/worker.py

```diff
@@ -119,6 +137,26 @@ def extract_stats(inp):
   return result
 
 
+def extract_ground_truth_stats(input_text):
+  """Returns a dictionary of ground truth stats."""
+  result = {}
+  parsed = GROUND_TRUTH_STATS_PATTERN.extract(input_text)
+
+  result['ground_truth_abs_trans_err'] = float(parsed['abs_trans_err'])
```

Comment:
Maybe make this a `for` loop over `abs_trans_err`, ...? Makes it more readable and more obvious you did not introduce typos. You could also extract a constant for this list of names to reuse.

Reply:
Done